.githooks/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+if [[ `git status --porcelain` ]]
+then
+  echo "un-commited changes found, do commit or stash them"
+  exit 1
+fi
```

Comment:
Not going to do anything else in pre-push?
Maybe we can shift `flutter analyze` step to here

Reply:
thought of running running ```flutter test``` in pre-push
but yeah will shift ```flutter analyse``` to pre-push from pre-commit